src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -714,13 +721,23 @@ private SyntaxTrivia VisitStructuredTrivia(SyntaxTrivia trivia)
             bool oldIsInStructuredTrivia = _isInStructuredTrivia;
             _isInStructuredTrivia = true;
 
-            SyntaxToken oldPreviousToken = _previousToken;
-            _previousToken = default(SyntaxToken);
+            SyntaxTrivia result;
+            if (trivia.IsSkippedTokensTrivia)
```

Comment:
Just for clarification, is this a behavior change? (I don't see usage of `IsSkippedTokensTrivia` in original code)

Reply:
I'm afraid I've completely forgotten what this code is doing by now, and it would take me a while to get back into it.

However it's not in the 2 behavioral changes I mentioned in the issue, so my guess is it was necessary to preserve behavior for some reason.

Reply:
Found the answer. I'd missed that `GetNextRelevantToken` was actually checking `IsSkippedTokensTrivia` before.